src/components/IspSelect/IspSelect.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../assets/variables";
+
+.IspSelect {
```

Comment:
I do prefer this way, but I think last project you suggested the dashes 😄  Let's switch em!